firefox_puppeteer/ui/windows.py

```diff
@@ -441,6 +441,9 @@ def __init__(self, *args, **kwargs):
         self._navbar = None
         self._tabbar = None
 
+        # Timeout for loading remote web pages
+        self.timeout_page_load = 30
```

Comment:
Hm, I totally missed that comment when landed the changes last Friday because nits haven't been mentioned on the Bugzilla review request. I will make sure to double check github first in the future before merging PRs. Sorry.